### DISCUSSION TITLE
Resource overspend with multiple Work Camps

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#518] [Ideas]
When building a Grand Temple, let's say I have supplied 10 out of 16 units for a given phase.
If there are at least 2 Work Camps active, both foremen will go looking for the missing resources, in some cases both coming back with 4 each.
The end result is 18/16 resources delivered.

These extra resources are gone.
Could they at least be carried over to the next phase? Phase 4 extra resources would be lost, yes, but at least not phases 1-3.

**Comment by Keriew** [comment]
The work camp foremen are supposed to be smart and coordinate the effort so no resources are wasted, but there were some oversights in the code. Fixed in d573779.

**Reply by catalin-duta** [reply]
Amazing, thank you!
